lambda/update/correction.go

```diff
@@ -201,12 +159,42 @@ func (c Correction) Apply(lpa *shared.Lpa) []shared.FieldError {
 	return nil
 }
 
+func (c Correction) Apply(lpa *shared.Lpa) []shared.FieldError {
+	if !c.SignedAt.IsZero() && !c.SignedAt.Equal(lpa.SignedAt) && lpa.Channel == shared.ChannelOnline {
```

Comment:
Is it possible to have this condition as a variable just so its quicker to read?